app/controllers/api/v2/hosts_bulk_actions_controller.rb

```diff
@@ -25,11 +26,64 @@ def bulk_destroy
         process_response @hosts.destroy_all
       end
 
+      api :PUT, "/hosts/bulk/build", N_("Build")
+      param_group :bulk_host_ids
+      param :reboot, :bool, N_("Reboot after build")
```

Comment:
If I'm reading correctly, this param is ignored if `rebuild_configuration` is true, right?
```suggestion
      param :reboot, :bool, N_("Reboot after build. Ignored if rebuild_configuration is passed.")
```